test/e2e/alphaupdate/update_test.go

```diff
@@ -105,6 +106,51 @@ var _ = Describe("kubebuilder", func() {
 			By("validating custom code preservation")
 			validateCustomCodePreservation(mockProjectDir)
 		})
+
+		It("should abort on merge conflicts when --force is not used", func() {
+			By("creating mock project with kubebuilder v4.5.2")
+			createMockProject(mockProjectDir, binFromVersionPath)
+
+			By("injecting conflicting code that will cause merge conflicts")
```

Comment:
Could we not generate exactly the same mock with 4.5.0 ?
It will generate a mock with conflicts
So that we do not need it.